templates/web/site-config.ftl

```diff
@@ -58,7 +59,13 @@
 
 </head>
 <body class="yui-skin-cstudioTheme">
-<div id="admin-console" class="categories-panel-active"></div>
+  <#if RequestParameters.mode?? && RequestParameters.mode == "embedded">
+    <#assign mode = RequestParameters.mode />
+  <#else>
+    <#assign mode = ""/>
+  </#if>
```

Comment:
You may want to put this above and, when mode is embedded, put some classes on the body and/or the root elem to target the sidebar & toolbar hide styles.

Reply:
@rart right now the class if being set to the `admin-console` root element, and styles applied to that class

Reply:
I moved the class up to the body (I think it's clearer)